### PR TITLE
feat: add footer

### PR DESCRIPTION
**Pull request by Purva2803**
### Description

(Write your answer here.)

### Checklist

- [ ] I am making a proper pull request, not spam.
- [ ] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

#179 

## Add relevant screenshot or video (if any)

(Add here)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,6 +70,15 @@ <h1 class="title">
   <a href="#" id="return-to-top" class="special1" > 
     <i class="fa fa-arrow-up" ></i>
   </a>
+  <!-- <div class="c1">
+    <lottie-player src="https://assets6.lottiefiles.com/packages/lf20_2scSKA.json"  background="transparent"  speed="1"  style="width: 400px; height: 400px;"  loop autoplay></lottie-player>
+  </div> -->
+  <footer>
+   <div class="foot">
+    Copyright © 2022 BookFinder
```
Comment (wasimreja) [inline index.html:78]:
```suggestion
    Copyright © 2022 <a href="https://github.com/wasimreja/book-finder">BookFinder</a>

```

File: index.html [hunk 2]
```diff
@@ -69,6 +70,15 @@ <h1 class="title">
   <a href="#" id="return-to-top" class="special1" > 
     <i class="fa fa-arrow-up" ></i>
   </a>
+  <!-- <div class="c1">
+    <lottie-player src="https://assets6.lottiefiles.com/packages/lf20_2scSKA.json"  background="transparent"  speed="1"  style="width: 400px; height: 400px;"  loop autoplay></lottie-player>
+  </div> -->
+  <footer>
+   <div class="foot">
+    Copyright © 2022 BookFinder
+    <div class="link"><a href="https://github.com/wasimreja/book-finder">Github</a></div>
```
Comment (wasimreja) [inline index.html:79]:
```suggestion

```

File: style.css [hunk 3]
```diff
@@ -295,6 +302,27 @@ h1 {
...
+	font-family : 'Comfortaa', cursive;
+	padding: 1rem 1rem;
+	background-color: #26272b;
+	color: #737373;
+	text-align: center;
+    transition: all 0.3s linear ;
+	
+
+}
+.link a{
```
Comment (wasimreja) [inline style.css:316]:
```suggestion
.foot a:hover {
    color: #bfbfbf;
    text-decoration: none;
    transition: all 0.3s linear;


```

**Review by wasimreja — CHANGES_REQUESTED** [review]
make the necessary changes

**Comment by Purva2803** [comment]
Yes i Worked on it just haven't created pull request will do by tonight.

File: index.html [hunk 4]
```diff
@@ -69,6 +70,17 @@ <h1 class="title">
   <a href="#" id="return-to-top" class="special1" > 
     <i class="fa fa-arrow-up" ></i>
   </a>
+  <!-- <div class="c1">
+    <lottie-player src="https://assets6.lottiefiles.com/packages/lf20_2scSKA.json"  background="transparent"  speed="1"  style="width: 400px; height: 400px;"  loop autoplay></lottie-player>
+  </div> -->
+  <footer>
+   <div class="foot">
+    <div class="link">
```
Comment (wasimreja) [inline index.html:78]: remove this
```suggestion


```

File: index.html [hunk 5]
```diff
@@ -69,6 +70,17 @@ <h1 class="title">
   <a href="#" id="return-to-top" class="special1" > 
     <i class="fa fa-arrow-up" ></i>
   </a>
+  <!-- <div class="c1">
+    <lottie-player src="https://assets6.lottiefiles.com/packages/lf20_2scSKA.json"  background="transparent"  speed="1"  style="width: 400px; height: 400px;"  loop autoplay></lottie-player>
+  </div> -->
+  <footer>
+   <div class="foot">
+    <div class="link">
+    Made with ❤️ by <a href="https://github.com/wasimreja">Wasim Raja</a>
```
Comment (wasimreja) [inline index.html:79]:
```suggestion
    Made with ❤️ by <a href="https://github.com/wasimreja">Wasim Reja</a>

```

File: style.css [hunk 6]
```diff
@@ -195,6 +195,13 @@ html {
 .remaining-description--show {
 	display: inline;
 }
+.c1{
```
Comment (wasimreja) [inline style.css:198]: remove this 
```suggestion
```

File: style.css [hunk 7]
```diff
@@ -295,6 +302,27 @@ h1 {
...
+    transition: all 0.3s linear ;
+	
+
+}
+.link a{
+	text-decoration: none;
+	color: white;
+
+}
+.link a:hover{
```
Comment (wasimreja) [inline style.css:321]: remove this
```suggestion

```

**Review by wasimreja — COMMENTED** [review]
update your branch with the main project

File: style.css [hunk 8]
```diff
@@ -295,6 +302,27 @@ h1 {
...
 	transition: all 0.3s ease;
 }
+.foot{
+
+	font-family : 'Comfortaa', cursive;
+	padding: 1rem 1rem;
+	background-color: #26272b;
+	color: #737373;
+	text-align: center;
+    transition: all 0.3s linear ;
```
Comment (wasimreja) [inline style.css:312]: remove this
```suggestion
```

File: style.css (hunk 3 above)
Comment (wasimreja) [inline style.css:316]: remove this 
```suggestion
```

File: style.css [hunk 9]
```diff
@@ -295,6 +302,27 @@ h1 {
 	-o-transition: all 0.3s ease;
 	transition: all 0.3s ease;
 }
+.foot{
+
+	font-family : 'Comfortaa', cursive;
+	padding: 1rem 1rem;
```
Comment (wasimreja) [inline style.css:308]:
```suggestion
	padding: 2rem 2rem;
```

**Comment by wasimreja** [comment]
@Purva2803 resolve conflicts

**Comment by Purva2803** [comment]
sorry @wasimreja for making you do this much efforts I am contributing for the first time so it's my learning phase there are the issues with branches so I am doing all the changes in the main and creating a pull request .

**Pull request closed without merging**